src/DynamoCore/Engine/Profiling/NodeProfilingData.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Dynamo.Core;
+using Dynamo.Graph.Nodes;
+using VMDataBridge;
+
+namespace Dynamo.Engine.Profiling
+{
+    class NodeProfilingData : NotificationObject, IDisposable
```

Comment:
is this our own `NotificationObject` class or interface - what is it doing here?

Reply:
It seems that this is our own notification item, but is not necessary here, has been removed.